controllers/users-controller.js

```diff
@@ -120,7 +120,19 @@ exports.loginUser = (appReq, appRes) => {
     if (err) return appRes.status(500).send('Error on the server.');
     if (!user) return appRes.status(404).send('No user found.');
 
-    const passwordIsValid = bcrypt.compareSync(
+    let passwordIsValid = bcrypt.compareSync(
+      appReq.body.password,
```

Comment:
everything is looking good but my one concern is here and below in reset password duplicating passwordIsValid